addons/website_sale/static/src/js/location_selector/map/map.js

```diff
@@ -62,7 +62,7 @@ export class Map extends Component {
             // Icon creation
             const iconInfo = {
                 className: String(loc.id) === this.props.selectedLocationId
-                           ? "o_location_selector_marker_icon_selected"
+                           ? "o_location_selector_marker_icon o_location_selector_marker_icon_selected"
```

Comment:
Isn't possible to have just one class here ?
```suggestion
                           ? "o_location_selector_marker_icon_selected"
```